### PR TITLE
added: feature to get historical option data

### PR DESCRIPTION
**Pull request by jitendravarma**
We have an requirment to seek historical option data for a given strike price, currently we are doing it by a custom script. So thought of extending the support to the library itself.

URL we are trying to mimic is : https://www.nseindia.com/report-detail/fo_eq_security

How is it tested?
On my local system for various strike price and option data

Let me know if this looks okay to you!

### CODE REVIEW

**Comment by jitendravarma** [comment]
Sample input: 
get_historical_option_data("TATAMOTORS", "25-04-2024", "26-04-2024","CE", "1020", "2024", "30-May-2024")

File: nsedt/derivatives/options.py [hunk 1]
```diff
@@ -114,3 +114,53 @@ def get_option_chain_expdate(symbol: str) -> list:
...
+        expiry_date (str): _description_.
+        response_type (str, optional): _description_. Defaults to "panda_df".
+        columns_drop_list (list, optional): _description_. Defaults to None.
+
+    Returns:
+        _type_: _description_
+    """
+    cookies = utils.get_cookies()
+    base_url = cns.BASE_URL
+    event_api = cns.FUTURES_PRICE
```
Comment (jitendravarma) [inline nsedt/derivatives/options.py:148]: Shall I rename this variable as it same for both futures and options.

Comment (jitendravarma) [reply nsedt/derivatives/options.py]: I have migrated the url to FNO_HISTORY and its reference.

File: nsedt/utils/data_format.py [hunk 2]
```diff
@@ -21,6 +21,7 @@ def price(result):
         "CH_TRADE_LOW_PRICE",
         "CH_CLOSING_PRICE",
         "CH_PREVIOUS_CLS_PRICE",
+        "CH_LAST_TRADED_PRICE",
```
Comment (jitendravarma) [inline nsedt/utils/data_format.py:24]: Add support for last price as its a very an important price for analysis.

**Review by pratik141 — APPROVED** [review]

**Pull request merged**